src/frontend/apps/desk/src/features/menu/MenuItems.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button } from '@openfun/cunningham-react';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import React, { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { Box } from '@/components/';
+import { useCunninghamTheme } from '@/cunningham';
+import { SVGComponent } from '@/types/components';
+
+import { Tooltip } from './Tooltip';
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #ffffff33;
+  &[aria-current='page'] {
+    color: #ffffff;
+  }
+`;
+
+interface MenuItemProps {
+  Icon: SVGComponent;
+  label: string;
+  href: string;
+  width?: number;
+}
+
+const MenuItem = ({ Icon, label, href, width = 38 }: MenuItemProps) => {
+  const { t } = useTranslation();
+  const pathname = usePathname();
+  const { colorsTokens } = useCunninghamTheme();
+  const buttonRef = useRef(null);
+  const [isTooltipOpen, setIsTooltipOpen] = useState(false);
+
+  const isActive = pathname === href;
+  const color = isActive ? colorsTokens()['primary-600'] : '#ffffff55';
+  const background = isActive ? colorsTokens()['primary-300'] : undefined;
+
+  return (
+    <StyledLink
+      href={href}
+      aria-current={isActive ? 'page' : undefined}
```

Comment:
```suggestion
      aria-current={isActive &&  'page'}
```

Reply:
It should work as `aria-current` accepts boolean. I don't know if there is a different semantic.

Reply:
https://github.com/numerique-gouv/people/commit/21d6524cd4a615e75a7776e66ee2c564dff0334d#diff-abc76eebde64ee4bb46549337a1c1a6cccc26482960132953ebb79940c3a2ad9R42